README.md

```diff
@@ -4334,13 +4334,14 @@ Using the preference as the key, set the `Value` to the corresponding preference
 
 `Status` can be "default", "locked", "user" or "clear"
 
-Default preferences can be modified by the user.
+* `"default"`: Read/Write: Setting appear as default even if factory default differs.
```

Comment:
Thanks for this!

i think we need an "s" on either the Setting or the appear on these three lines.